src/frontend/assets/index.html

```diff
@@ -4,10 +4,8 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta
-      http-equiv="Content-Security-Policy"
-      content="CSP_POLICY_PLACEHOLDER"
-    />
+    <!-- CSP directives injected by the canister -->
+    <meta replaceme-with-csp />
```

Comment:
just in case, I guess it can even be a comment

```
<!-- replaceme-with-csp -->
```

```
let index_html = index_html.replace("<!-- replaceme-with-csp -->", &format!(r#"<meta http-equiv="Content-Security-Policy" content="{}" />"#,&http::content_security_policy_meta() ));
```

Reply:
Comments will be stripped by webpack when minimizing the HTML I think

Reply:
Ah good to know, isn't the case with Rollup out-of-the-box. Thanks for the feedback.